lib/utils/ProductListQueryConfiguration.dart

```diff
@@ -12,8 +12,6 @@ class ProductListQueryConfiguration extends AbstractQueryConfiguration {
     this.barcodes, {
     final OpenFoodFactsLanguage? language,
     final List<OpenFoodFactsLanguage> languages = const [],
-    @Deprecated('Use parameters language or languages instead')
-        final String? lc,
     @Deprecated('Use parameter country instead') final String? cc,
```

Comment:
cc

Reply:
`cc` is much younger, it's from 2021-11-15 (2021-07-20 for `lc`).